packages/components/psammead-play-button/src/index.jsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import { string, oneOf, func } from 'prop-types';
+import styled, { css } from 'styled-components';
+import { C_EBON, C_WHITE, C_POSTBOX } from '@bbc/psammead-styles/colours';
+import { GEL_SPACING, GEL_SPACING_TRPL } from '@bbc/gel-foundations/spacings';
+import { GEL_MINION } from '@bbc/gel-foundations/typography';
+import { getSansBold } from '@bbc/psammead-styles/font-styles';
+import { mediaIcons } from '@bbc/psammead-assets/svgs';
+import VisuallyHiddenText from '@bbc/psammead-visually-hidden-text';
+
+const GEL_SPACING_DEC = '5rem';
+const BGC_TRANSITION_DURATION = '300ms';
+
+const Button = styled.button`
+  background-color: ${C_EBON};
+  border: none;
+  color: ${C_WHITE};
+  cursor: pointer;
+  display: block;
```

Comment:
I think this may need
```
possition: absolute;
bottom: 0;
```
To position it at the bottom of the player placeholder :)

Reply:
I've been wondering about this. In the 'in situ' GIF 👆, I used the parent to position it. The button knows which service it's rendering in, so _could_ position itself (e.g. for RTL), but my concern is that this would limit its use. What do you think?

Reply:
I think if you only specify `bottom` and not `left` or `right`, the browsers default `rtl`/`ltr` behaviour will do that for us :) 

![image](https://user-images.githubusercontent.com/11341355/65997297-05310400-e491-11e9-821e-6fe73c9d20b8.png)

If you select `persian` in storybook and add these styles, it works with `rtl` automatically :D

Reply:
That's awesome. I'll do that. Thanks, Drew!

Reply:
Hey @dr3, I've pushed a few changes that relinquish some style control to the parent following our discussion earlier.